allure/utils.py

```diff
@@ -160,3 +160,10 @@ def host_tag():
     Return a special host_tag value, representing current host.
     """
     return socket.gethostname()
+
+def clean_docstring(docstring):
+    if docstring is not None:
+        docstring = docstring.replace('\n    ', '\n')
+        return(docstring)
+    else:
+        return ''
```

Comment:
Should find out why our static checker wont complain about `no newline at the end of file`, but plz fix.